src/app.js

```diff
@@ -1 +1,18 @@
 'use strict';
+
+const fs = require('fs');
+const datas = process.argv.slice(2);
+
+if (datas[0] === 'cp') {
```

Comment:
Add a check to ensure that the source and destination files are not the same before calling the 'copyFile' function.